BlueprintUI/Tests/ElementStateTests.swift

```diff
@@ -32,6 +32,14 @@ extension ElementStateTree {
 
 class ElementStateTreeTests: XCTestCase {
 
+    override func setUp() {
```

Comment:
I wonder if we should add a test base class, and run each test twice; once with caching on, and once with it off?

Reply:
(Eg, override invoke test:)

```
override func invokeTest() {
   BlueprintView.enableCrossLayoutCaching = true
   super.invokeTest()

   BlueprintView.enableCrossLayoutCaching = false
   super.invokeTest()
}
```

Reply:
Did you mean for all the touched tests? This won't work for the `ElementStateTree` tests since some will break. We can do it for the other tests though.